topic-operator/src/main/java/io/strimzi/operator/topic/Config.java

```diff
@@ -106,7 +106,8 @@ private Value(String key, Type<? extends T> type, boolean required) {
     public static final String TC_TLS_TRUSTSTORE_PASSWORD = "STRIMZI_TRUSTSTORE_PASSWORD";
     public static final String TC_TLS_KEYSTORE_LOCATION = "STRIMZI_KEYSTORE_LOCATION";
     public static final String TC_TLS_KEYSTORE_PASSWORD = "STRIMZI_KEYSTORE_PASSWORD";
-
+    public static final String TC_TLS_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG = "STRIMZI_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG";
```

Comment:
Do we need the `_CONFIG` at the end of the variable as well as value? None of the other options does have it, so I think it would be better to be consistent.

Reply:
removed